src/styles/Icons.jsx

```diff
@@ -16,7 +16,7 @@ const ClockIcon = (props) =>
 
 const DollarIcon = (props) =>
   <SvgIcon {...props}>
-    <path d="M11.8 10.9c-2.27-.59-3-1.2-3-2.15 0-1.09 1.01-1.85 2.7-1.85 1.78 0 2.44.85 2.5 2.1h2.21c-.07-1.72-1.12-3.3-3.21-3.81V3h-3v2.16c-1.94.42-3.5 1.68-3.5 3.61 0 2.31 1.91 3.46 4.7 4.13 2.5.6 3 1.48 3 2.41 0 .69-.49 1.79-2.7 1.79-2.06 0-2.87-.92-2.98-2.1h-2.2c.12 2.19 1.76 3.42 3.68 3.83V21h3v-2.15c1.95-.37 3.5-1.5 3.5-3.55 0-2.84-2.43-3.81-4.7-4.4z" />
+    <path d="M12.011,20v-3.619H8.752v-1.313h3.259v-1.444H8.752v-1.313h2.611L7.895,6h2.779l1.755,3.899 c0.376,0.797,0.606,1.464,0.877,2.153h0.042c0.25-0.646,0.522-1.4,0.877-2.197L16.043,6h2.717l-3.656,6.311h2.59v1.313h-3.28v1.444 h3.28v1.313h-3.28V20H12.011z" />
```

Comment:
@bill-feng !!! nice! Where did you find this path for yen icon? 👍

Reply:
Online! :D